### DISCUSSION TITLE
SCM with Ocean Model

### DISCUSSION THREAD

**Discussion by dustinswales** [NCAR/ccpp-scm#395] [Ideas]
Recently there has been some interest in adding the ability to couple the SCM with a simplified(?) ocean model. 
From a technical perspective the pathway to implementation is ~straightforward, and could be accomplished by providing the state to the ocean model somewhere in [scm_time_integration.F90](https://github.com/NCAR/ccpp-scm/blob/main/scm/src/scm_time_integration.F90). 

@grantfirl You have more experience with the SCM, but "I think" if we build a component ocean model as part of the SCM build, then can we call it in scm_time_integration.F90? 
@yangfanglin I'm out of my lane when it comes to ocean models, so I can't speak to what the interface would look like, but conceptually this appears to be feasible.

**Comment by yangfanglin** [comment]
@dustinswales Thanks for picking up the conversation we had at the UIFCW meeting last week.  The coupling can be made to a simple 1-D mixed layer ocean model or a more complete ocean model like MOM6 if a 1-D version exists.   I have a 1-D mixed-layer ocean model of intermediate complexity https://www.emc.ncep.noaa.gov/gmb/wx24fy/fyang/www/modexp/mlo.html.   For forecasts at MRW/S2S time scale, the relaxation of (T,S) to climatology and horizontal advection terms can even be turned off /ignored.  This is one option and it is fine to use any other mixed layer ocean model.
